README.md

```diff
@@ -80,3 +83,32 @@ To use **rticles** from RStudio:
     rmarkdown::draft("MyJSSArticle.Rmd", template = "jss_article", package = "rticles")
     rmarkdown::draft("MyRJournalArticle", template = "rjournal_article", package = "rticles")
     ```
+
+## How to contribute ? 
+
+Most of the templates are contributed directly by the users in the community. 
+If you want `rticles` to offer a new journal format, you can contribute by the following way.
```

Comment:
1. Github turns a line break in Markdown to `<br/>` (unlike Pandoc, which ignores line breaks by default), which we don't want.
1. We use bold text for **package** names.

```suggestion
Most of the templates are contributed directly by the users in the community. If you want **rticles** to offer a new journal format, you can contribute by the following way.
```

Similar issues in the rest of text below.